### DISCUSSION TITLE
Node Version

### DISCUSSION THREAD

**Discussion by tdashworth** [qetza/replacetokens-task#59] [Q&A] [answered]
Hi,

Just double checking here. I see only Node version 16 is supported but the Microsoft Hosted Agents have version 18 installed. 

Will that be a problem?

**Comment by tdashworth** [comment, marked as answer]
Tested in my own org. Seems for work on Windows and Ubuntu.

**Comment by qetza** [comment]
Hi @tdashworth,
Node 16 is the minimum version required so it will work with agent having node 18.
